### DISCUSSION TITLE
Hello

### DISCUSSION THREAD

**Discussion by seanmsimon82** [community/community#119663] [New to GitHub]
I have lost access to my account, both my password and my 2FA are lost, as I lost access to Gmail and my cellphone.  Is there a way that I can recover my old account?

**Comment by eliz-amst** [comment]
Open ticket with GitHub support, try resetting the phone number, double check if you can get access to your gmail account

**Comment by niladri-1** [comment]
Hey there I hope these links work for your solution.

https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials

**Comment by cyrolux123** [comment]
May be this can help https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials

**Comment by seanmsimon82** [comment]
I don’t have a recovery code, it got lost.  I also don’t have a security
key or fallback number or a verified device, or SSH token, or a personal
access token.  Is there some other way that I can get access to the account?

The account is https://github.com/seanmsimon/

**Comment by FrootLoopsBowl** [comment]
I'm sorry to inform you that your account is permanently lost.
